lib/cn/kvset.c

```diff
@@ -135,7 +135,7 @@ lvx2vbd(struct kvset *ks, uint i)
 }
 
 static void
-_kvset_destroy(struct kvset *ks);
+_kvset_close(struct kvset *ks);
```

Comment:
Would prefer you rename this to kvset_close_impl() while you're at it...

Reply:
My bad, just kvset_close() seems fine..

Reply:
I have seen @nabeelmmd removing prefixed underscores for private functions, so I can do the same here and wherever else I touch.